### PR TITLE
Breadcrumb doc

### PR DESCRIPTION
**Pull request by sidra2022**

### CODE REVIEW

File: packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.tsx [hunk 1]
```diff
@@ -18,10 +18,11 @@ export class DatacomBreadcrumb {
     return (
       <Host>
         {!!this.url ? (
-          <div>
-            <a href={this.url} class="dc-breadcrumb">
-              <slot></slot> {this.NavIcon}
+          <div class="dc-breadcrumb">
+            <a href={this.url} class=" dc-text">
```
Comment (AngeCast) [inline packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.tsx:22]: You got an empty space before your class here. Please remove for code cleanliness.

File: packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.css [hunk 2]
```diff
@@ -17,7 +17,37 @@
...
+  transform: scaleX(0);
+  width: 0;
+  height: 1px;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  transition: transform 0.25s ease-out;
+}
+
+a.dc-text:hover::before {
```
Comment (AngeCast) [inline packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.css:41]: Do you actually need the 'a' tag selector? Couldn't you just use the .dc-text class you've introduced?

Just an FYI in the future, try to keep your selectors as short as possible. It just helps with code readability and maintenance.

File: packages/stencil/src/components/datacom-breadcrumb/usage.md [hunk 3]
```diff
@@ -3,9 +3,14 @@
 Breadcrumb has one component 'datacom-breadcrumb'
 ### Properties - datacom-breadcrumb
 
--url for passing link of other page. Styled with 'dc- breadcrumb' class.
+-url for passing link of other page.
```
Comment (AngeCast) [inline packages/stencil/src/components/datacom-breadcrumb/usage.md:6]: Having a hard time understanding what you're trying to say with these documentation updates here. Might want to revise your grammar here.

**Review by AngeCast — CHANGES_REQUESTED** [review]

File: packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.tsx (hunk 1 above)
Comment (sidra2022) [reply packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.tsx]: Done

File: packages/stencil/src/components/datacom-breadcrumb/usage.md [hunk 4]
```diff
@@ -3,9 +3,13 @@
 Breadcrumb has one component 'datacom-breadcrumb'
 ### Properties - datacom-breadcrumb
 
--url for passing link of other page. Styled with 'dc- breadcrumb' class.
+-Component  is render separator (>) only when it is wrap with 'div' .
```
Comment (AngeCast) [inline packages/stencil/src/components/datacom-breadcrumb/usage.md:6]: My previous comment was for ALL the new documentation you added, not just the one line. It's still not too clear what you're trying to convey.

**Review by AngeCast — CHANGES_REQUESTED** [review]

File: packages/stencil/src/components/datacom-breadcrumb/usage.md [hunk 5]
```diff
@@ -1,16 +1,23 @@
...
+  <datacom-breadcrumb url="https://datacom.com/nz/en/discover" separator>Breadcrumb</datacom-breadcrumb>
+  <datacom-breadcrumb>Breadcrumb</datacom-breadcrumb>
+</div>
+```
+### Properties - datacom-breadcrumb
+
+url-This is the url to the page where breadcrumb link redirects to.
+separator- This is prop which is used to show Icon(>) alongside breadcrumb text.
+
+"If no url is provided, the component will be rendered as plain text."
```
Comment (AngeCast) [inline packages/stencil/src/components/datacom-breadcrumb/usage.md:23]: You should include this line with the url prop description.

**Review by AngeCast — APPROVED** [review]

**Review by AngeCast — CHANGES_REQUESTED** [review]
One more change to your documentation.

File: packages/stencil/src/components/datacom-breadcrumb/usage.md (hunk 5 above)
Comment (AngeCast) [reply packages/stencil/src/components/datacom-breadcrumb/usage.md:23]: Sorry, didn't mean to approve this yet.

Comment (sidra2022) [reply packages/stencil/src/components/datacom-breadcrumb/usage.md]: No worries, thanks for spotting it.  I fixed it.

**Review by AngeCast — APPROVED** [review]

**Pull request merged**